zwave_js_server/model/node/__init__.py

```diff
@@ -670,16 +670,19 @@ async def async_set_location(
         )
         self.data["location"] = location
 
-    async def async_get_firmware_update_progress(self) -> bool:
+    async def async_get_firmware_update_progress(self, any_node: bool = False) -> bool:
```

Comment:
I know. Do you have alternate suggestions? Right now it's only the node message handler that has access to this information. I suppose we could give the controller message handler access to this information so that we can make a separate controller command. @balloob any thoughts on this?

Reply:
yeah that seems logical to me. I was hoping I could squeeze this in to that last addon update but better to do this correctly. We can always do a new schema for this change so we can force support for it. This would ideally be a part of the firmware update feature to improve the experience.